### DISCUSSION TITLE
Circle and Cylinder sinks into Plane

### DISCUSSION THREAD

**Discussion by MorningLit** [pmndrs/use-cannon#431] [Q&A] [answered]
Hi all, not sure if this is normal for this library or it's my code, but I am trying to make a coin-flipping website and I am not sure why the coin is "sunk" or "dug" into the plane.
Example:
![image](https://user-images.githubusercontent.com/57279935/235130189-612804e7-7093-433a-8c9a-93cc64a336c3.png)
![image](https://user-images.githubusercontent.com/57279935/235130755-8bd7cfd4-6a6f-4c61-8ba6-5f334ce04711.png)

I have read this issue but it doesn't seem to address it https://github.com/pmndrs/use-cannon/issues/58
I tried both circleGeometry and cylinderGeometry but both produce the same results. Is this the supposed default behaviour that I am supposed to be seeing?

**Comment by isaac-mason** [comment]
It's hard to provide help with what you've shared. Could you create a codesandbox so we can see what you're doing?

Also, have you tried using the Debug component to see the physics colliders?

**Reply by MorningLit** [reply, marked as answer]
Thanks for the prompt reply! After playing around for a bit, I realised I used useBox instead of useCylinder 😞 
Apologies, new to this library

**Reply by isaac-mason** [reply]
No worries! If you're after some more help, check out the discord: https://pmnd.rs/discord

There's a channel there for use-cannon where myself and others are happy to help :)
